packages/docs/src/routes/docs/cookbook/detect-img-tag-onload/index.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: Cookbook | Detect img tag onLoad
+contributors:
+ - gioboa
+---
+
+import CodeSandbox from '../../../../components/code-sandbox/index.tsx';
+
+
+# Detect img tag onLoad even if it's cached
+
+When using the onLoad$ property on an \<img\> tag in a Qwik component, you might encounter an issue where the onLoad$ event doesn't fire when the image is already cached in the browser.<br/>
+In that case the onLoad$ logic is not being triggered.<br/>
+This behavior is observed during normal navigation, but not in "private window" sessions or when browser caching is disabled, indicating the cache's impact on the event.<br/>
```

Comment:
> but not in "private window" sessions
It's called "incognito navigations" in English

But if you navigate in incognito navigation, and you cache is **not** disabled, then you will get the cache issue. So I think you can just remove that part.